apps/sms/views/conversation/test/unit/conversation_test.js

```diff
@@ -6786,6 +6783,8 @@ suite('conversation.js >', function() {
         ConversationView.initRecipients();
         Navigation.isCurrentPanel.withArgs('composer').returns(true);
         this.sinon.stub(ConversationView.recipients, 'focus');
+        this.sinon.stub(Compose, 'focus');
+        this.sinon.stub(ConversationView, 'hasValidRecipients');
```

Comment:
Could you elaborate why it is not valid? What do you mean by that?

Actually here, as we are unit-testing, isn't it better to treat results of other functions as part of the external state, and therefore to mock them, just like parameters? That's usually how I write my unit test (but are we really unit-testing here?) . WDYT?